src/pages/messages/components/DeleteIconButton.tsx

```diff
@@ -0,0 +1,32 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import { IconButton, IconButtonProps, SxProps, Tooltip } from '@mui/material';
+import { mainTheme } from '../../../themes/mainTheme';
+
+interface Props extends IconButtonProps {
+  iconSx?: SxProps | undefined;
```

Comment:
Ser ikke ut som denne brukes? Syns det er greit å vente til det skal brukes for å unngå at ting "bloates" pga ting som kanskje trengs en dag
```suggestion
```